rps.js

```diff
@@ -0,0 +1,54 @@
+const playGame= function(userChoice)
+{
```

Comment:
issues with the out, the output should match the readme "Computer picked: paper. User picked: rock. User lose" your output only prints user "user tie", "user Lose", or "User win"